swagger-spec/users/external_identities_list.yaml

```diff
@@ -0,0 +1,43 @@
+get:
+  summary: List all user's external identities
```

Comment:
For consistency, would suggest "List all external identities" (the summary for listing user preprints, for example, is "List all preprints")